README.md

```diff
@@ -5,9 +5,10 @@ Small repo done in cab ride
 A simple repo used to track progress of codechef monthly contests from JAN15-JUNE17 (Can be increased or decresed by modifying code)
 
 ## What to do?
-* Install virtual env
+* Install virtual env `sudo apt-get install virtualenv`
 * Create an environment using virtual env `virtualenv envName -p python3`
-* Activate the environment and install the requirements `source bin/activate; pip3 install -r req.txt`
+* Activate the environment and install the requirements `source envName/bin/activate; pip3 install -r req.txt`
 * Run helper.py, wait for it to complete
+* Create a user using `python manage.py createsuperuser`
 * Start the server `python manage.py runserver`
-* Go to the said browser and port, only single URL is configured Eg: localhost:8000/users/pandusonu
+* Go to the said browser and port, only single URL is configured Eg: localhost:8000/user/superuser_name
```

Comment:
Not the superuser name, should be the name of the codechef profile you want to check